browser/ui/brave_file_select_utils.cc

```diff
@@ -46,41 +46,19 @@ url::Origin UnwrapOriginIfOpaque(const url::Origin& origin) {
 
 std::u16string GetFileSelectTitle(content::WebContents* web_contents,
                                   const url::Origin& alerting_frame_origin,
+                                  const url::Origin& download_origin,
                                   FileSelectTitleType file_select_type) {
-  // This implementation partially mirrors
-  // ChromeAppModalDialogManagerDelegate::GetTitle().
-  // TODO(sko) It's hard to test this behavior is in sync at this moment. Even
-  // upstream tests aren't covering this. Need to figure out how we can test
-  // extension and isolated web app case.
   CHECK(web_contents);
-  Profile* profile =
-      Profile::FromBrowserContext(web_contents->GetBrowserContext());
-
-  UrlIdentity url_identity = UrlIdentity::CreateFromUrl(
-      profile, alerting_frame_origin.GetURL(),
-      /*allowed_types*/
-      {UrlIdentity::Type::kDefault, UrlIdentity::Type::kFile,
-       UrlIdentity::Type::kIsolatedWebApp, UrlIdentity::Type::kChromeExtension},
-      /*default_options*/ {.default_options = {}});
-
-  if (url_identity.type == UrlIdentity::Type::kChromeExtension) {
-    return url_identity.name;
-  }
-
-  if (url_identity.type == UrlIdentity::Type::kIsolatedWebApp) {
-    return url_identity.name;
-  }
-
```

Comment:
took out the code above because i'm not sure we actually need it

* it only returns the name of an extension/app, not a proper message like "$app wants to download..."
* i think the [chromium](https://source.chromium.org/chromium/chromium/src/+/main:components/javascript_dialogs_strings.grdp;l=18?q=IDS_JAVASCRIPT_MESSAGEBOX_TITLE_EXTENSION&start=11) version is just intended to return a human-friendly name for extensions/etc. it's nice to have but without it, i think we would just show the generic "this page" message, which is ok.

Reply:
this is how it looks with this change
![image](https://github.com/user-attachments/assets/283edcde-1f75-4259-8315-7da4a04978f6)

Reply:
@goodov thanks for testing it out. i think we can live with that and open a follow up unless you want to address it in this PR?

Reply:
security wise i think this view is better because an extension/app could name themselves "google.com" or something

Reply:
would be nice to have it like `Extension <...> wants to save`. For consistency a similar change should be done for "wants to open" dialog and also for apps(?). A follow up sounds reasonable here.

Reply:
opened https://github.com/brave/brave-browser/issues/42987